dev/CommonStyles/APITests/CommonStylesTests.cs

```diff
@@ -0,0 +1,237 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Common;
+using MUXControlsTestApp.Utilities;
+using System.Linq;
+using System.Threading;
+using Windows.Foundation.Metadata;
+using Windows.System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using MUXControls.TestAppUtils;
+using PlatformConfiguration = Common.PlatformConfiguration;
+using OSVersion = Common.OSVersion;
+using System.Collections.Generic;
+
+#if USING_TAEF
+using WEX.TestExecution;
+using WEX.TestExecution.Markup;
+using WEX.Logging.Interop;
+#else
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.Logging;
+#endif
+
+
+
+namespace Windows.UI.Xaml.Tests.MUXControls.ApiTests
+{
+    [TestClass]
+    public class CommonStylesVisualTreeTests: VisualTreeTestBase
```

Comment:
Strongly recommend not requiring a base class for tests in order to do visual comparisions.

Reply:
VisualTreeTestBase  is removed.